packages/react/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  c,
+  classy,
+  cssVars,
+  m,
+  ProgressProps as BaseProps,
+} from '@onfido/castor';
+import React from 'react';
+
+export const Progress = ({
+  value,
+  min = 0,
+  max = 100,
+  size = 'regular',
+  style = {},
+  hideLabel,
+  children,
+  className,
+  'aria-valuetext': ariaValuetext,
+  ...restProps
+}: ProgressProps): JSX.Element => {
+  const percentValue = `${Math.round(((value - min) * 100) / (max - min))}%`;
+
+  return (
+    <div
+      {...restProps}
+      className={classy(c('progress'), m(size), className)}
+      role="progressbar"
+      aria-valuenow={value}
+      aria-valuemin={min}
+      aria-valuemax={max}
+      aria-valuetext={
+        ariaValuetext || (typeof children === 'string' ? children : undefined)
+      }
+      style={{ ...style, ...cssVars({ percentValue }) }}
```

Comment:
This will break if `style = null`.

I would suggest to not default to `{}` upon prop destruction, but instead:

```suggestion
      style={{ ...(style ?? {}), ...cssVars({ percentValue }) }}
```

Reply:
That is not true, `null` can be spread just fine, as can any other type.

![image](https://user-images.githubusercontent.com/18623773/122543370-9e1c0000-d023-11eb-9dfc-e5c5a97002a3.png)

We don't need either.